### DISCUSSION TITLE
Intel Arc (randomly) Device not Found

### DISCUSSION THREAD

**Discussion by ethen-sanchez** [vladmandic/sdnext#2517] [Q&A]
So every once in a while while running stable diffusion, it'll produce an error. 
99% of that time it has the error it'll be "Native API failed. Native API returns: -1 (PI_ERROR_DEVICE_NOT_FOUND)"; I honestly can't remember the other error as it happens so rarely. 
It happens randomly. It can happen on batch 1 or batch 100. 
I'm running via the .\webui.bat --use-ipex. I've tried --debug but it doesn't really give me any additional information. 
I have seen that some people recommended removing the config.json and restarting as it was using CPU instead of GPU so I tried that and it still happens. When i finish an image or batch it does show memory={'ram': ##/63.8, 'gpu': ##/15.56} so I'm assuming it's using the GPU correctly. 

It's not a huge deal as it just randomly happens, but I'm curious to see if other people have had the issue or have been able to fix it.

**Comment by Disty0** [comment]
That's basically a driver crash. What is your driver version? Minimum supported is 4952.

**Reply by ethen-sanchez** [reply]
I had an older version of Automatic. This was happening in on 4676. 
Before posting the original, I saw that there had been updated documentation to the install process (mainly that it now supported 4952 and higher). I did a git pull and updated my driver. The newest update from Intel is 4953 and that's what I'm using right now. I downgraded to 4952 as well but still the same. I'm back on 4953.
